rules-tests/Renaming/Rector/FileWithoutNamespace/PseudoNamespaceToNamespaceRector/Fixture/phpunit_test_case_split.php.inc

```diff
@@ -2,7 +2,7 @@
 
 namespace Rector\Tests\Renaming\Rector\FileWithoutNamespace\PseudoNamespaceToNamespaceRector\Fixture;
 
-class Fixture6
+class PhpunitTestCaseSplit
```

Comment:
Do we need a new phpstan rule which males sure classes within fixtures are uniquely named?

Reply:
probably needed , sometime, phpstan still read previous class "in memory", probably as "analyzed file" cache